README.md

```diff
@@ -67,6 +69,43 @@ and form generation using an [LLM](https://en.wikipedia.org/wiki/Large_language_
     docker run -p 5000:5000 fertiscan-backend
     ```
 
+#### Docker Compose
+
+1. Create a `.env` file from [.env.template](./.env.template). Include the
+   following environment variables:
+
+    ```ini
+    BB_URL=bytebase_url
+    BB_SERVICE_ACCOUNT=your-bytebase-sa@service.bytebase.com
+    BB_SERVICE_KEY=your-bytebase-sa-key
+    BB_INSTANCE_ID=your-bytebase-instance-id
+    BB_DATABASE_ID=your-bytebase-database-id
+    ```
+
+    You can find their values in our vault under fertiscan-dev.
+
```

Comment:
There should be a mention that the `FERTISCAN_DB_URL` env var needs to point to the `postgres` service. In this particular case: `postgresql://postgres:postgres@postgres:5432/fertiscan`.

Reply:
I updated the readme